io.github.MrPiggy92.ScriptedJourneys.json

```diff
@@ -0,0 +1,36 @@
+{
+  "id": "io.github.MrPiggy92.ScriptedJourneys",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "wrapper.sh",
+  "finish-args": [
+    "--filesystem=home",
```

Comment:
why is home needed?

Reply:
It needs to make a config folder in .config. Should I specify the config folder that it will create on first run or just home/.config?

Reply:
No it should use $XDG_CONFIG_HOME to create the config folder, then no permission is necessary.